consul/consul_client_03.md

```diff
@@ -0,0 +1,28 @@
+# Be a good Consul client
+
+Consul servers may be distributed but their numbers is limited (we use cluster of 5 per datacenter). And if you are not careful, Consul requests may all be forwarded to the cluster's leader. On the other hand, you loose a lot of Consul's benefits if you restrict too much the calls to Consul.
```

Comment:
Add a very small intro describing what you will discuss in the article to give some context